db_classes1.py

```diff
@@ -142,15 +137,41 @@ def __init__(self, ID, start_date, finish_date, employee_id, database):
 
     def save(self):
         self.sql_script = f'''INSERT INTO DEPARTMENT (ID, start_date, finish_date, employee_id) 
-            VALUES ('{self.ID}', '{self.start_date}', '{self.finish_date}', '{self.employee_id}')'''
+            VALUES ('{self.id}', '{self.start_date}', '{self.finish_date}', '{self.employee_id}')'''
+        employees_on_vacation = f'''?????????????'''
+        if employees_on_vacation >= 5:
+            return print('Currently on vacation are 5 employees')
         return self.database.save(self.sql_script)
 
     def get(self, *args, **kwargs):
-        self.sql_script = f'''SELECT {args} FROM department'''
+        self.sql_script = f'''SELECT {args} FROM HistoryVacation'''
         return self.database.get(self.sql_script)
 
-    def check_number_employees_vacation(self):
-        pass
 
+class Company:
+    def __init__(self, employee, position):
+        self.employee = employee
+        self.position = position
 
-db = Database('factory')
+    def get_salary(self):
```

Comment:
This method should get `employee` as a parameter. Or `employee_id` and then find the specific employee in the Employee list.